README.md

```diff
@@ -91,6 +92,7 @@ Note: Our setup script does these two for you, but they are here for reference.
     ```bash
     ./setup.sh
     ```
+1. Open Docker Desktop (Works for Windows, macOS, and Linux) or run Docker Daemon
```

Comment:
Thanks for the update, looks good.

Can you also move this line before the setup.sh?

So it should be:
1. clone the repo
2. run docker desktop
3. setup.sh 
4. ...
5. ...
6. ...